packages/@aws-cdk/aws-synthetics/README.md

```diff
@@ -44,7 +44,7 @@ const canary = new synthetics.Canary(this, 'MyCanary', {
     code: synthetics.Code.fromAsset(path.join(__dirname, 'canary')),
     handler: 'index.handler',
   }),
-  runtime: synthetics.Runtime.SYNTHETICS_NODEJS_2_2,
+  runtime: synthetics.Runtime.SYNTHETICS_NODEJS_PUPPETEER_3_0,
```

Comment:
Why change this value in all README examples? is this the recommended value by Synthetics?

Reply:
`syn-nodejs-2.2` runtimes are flagged with the following upgrade notice in the AWS Console:
![synthetics-2 2-notice](https://user-images.githubusercontent.com/749101/107840558-6c89f280-6dff-11eb-98a9-a89ba438cca3.png)